src/app/_types/sortTypes.ts

```diff
@@ -23,7 +23,7 @@ export type SortOption = {
 }
 
 export type SortableByDate = {
-  publishedOn?: DynamicBaseData['publishedOn']
+  publishedOn?: DynamicBaseData['published-on']
```

Comment:
Should we need to use kebab-case here?

Reply:
I think so because only `published-on` exists on `DynamicBaseData` now right?

The conversion to camel case only happens once the data goes through the helper function.

Either way, the data type we get should be the same so not a problem.

Reply:
@CharlyMartin 💯